applications/PoromechanicsApplication/CMakeLists.txt

```diff
@@ -69,9 +72,7 @@ if(${USE_PORO_MPI} MATCHES ON)
         ${CMAKE_CURRENT_SOURCE_DIR}/custom_conditions/surface_normal_fluid_flux_3D_diff_order_condition.cpp
     )
     
-    ## define library Kratos which defines the basic python interface
-    add_library(KratosPoromechanicsApplication SHARED ${KRATOS_POROMECHANICS_APPLICATION_SOURCES})
-    target_link_libraries(KratosPoromechanicsApplication KratosCore KratosTrilinosApplication KratosSolidMechanicsApplication)
+    #target_link_libraries(KratosPoromechanicsApplication KratosCore KratosTrilinosApplication KratosSolidMechanicsApplication)
```

Comment:
you should REALLY try to avoid such dependencies. 
having said this, take a look to the conversation in the pybind branch on how this should be handled in the StructuralMechanics and FluidMechanics

Reply:
Yeah, I know... I have double dependency when using MPI (Trilinos and SolidMech.), but for Serial/OMP I have only one dependency...